airflow/utils/log/es_task_handler.py

```diff
@@ -53,9 +53,11 @@ class ElasticsearchTaskHandler(FileTaskHandler, LoggingMixin):
     PAGE = 0
     MAX_LINE_PER_PAGE = 1000
 
+    # 16 is reasonable in this case
+    # pylint: disable-msg=too-many-arguments
     def __init__(self, base_log_folder, filename_template,
                  log_id_template, end_of_log_mark,
-                 write_stdout, json_format, json_fields,
```

Comment:
we may want to add a default value of `index` to be backward compatible.

Reply:
the default right now is "*" which has the same semantic as before and it is backward compatible.

Reply:
i mean a default value in the class constructor. although you have added an index in the airflow_local_settings, users might already have their own customized airflow_local_settings. If you don't put a default value in the index, it will break their code as the constructor requires `index`.

Reply:
done. Fixed as you suggested. Thanks.